### ISSUE TITLE
Cite.jl integration

### ISSUE THREAD

**Issue by pfitzseb** [JuliaComputing/JuliaHub#119]
Should generate a bibtex file and quotation sentence with [Cite.jl](https://github.com/SebastianM-C/Cite.jl) during docsgen. Can then add a button to the sidebar to download that file and copy the sentence to the clipboard.

**Comment by SebastianM-C** [comment]
Right now, Cite.jl allows the output of `get_tool_citation` to be printed to a given `IO`. Is that ok, or would you need something else?
I'm not sure if it would be relevant, but there is also an option to select only direct dependencies (with `only_direct=true`), which in the `docs` environment would be only the current package.

**Comment by pfitzseb** [comment]
Yeah, that's fine. I'd kinda like a way to run Cite.jl functions in an arbitrary dir independently of the active environment, but that's not critical.

**Comment by SebastianM-C** [comment]
I wanted to have that functionality in Cite.jl too, but I don't know how to use `Pkg.dependencies()` like that.
